rpc/wrtc_call_queue_mongodb.go

```diff
@@ -422,14 +424,13 @@ func (queue *mongoDBWebRTCCallQueue) changeStreamManager() {
 		}
 	}()
 	var lastSeq uint64
-	var ranOnce bool
+	var ranOnce bool // this is only for the first time the changestream is setup
```

Comment:
[minor] would it be worth renaming to `initialized`? or something else that documents that it only happens once.

Reply:
changed to isInitialized